google-account-plugin/src/com/google/cloud/tools/intellij/stats/GoogleUsageTracker.java

```diff
@@ -81,17 +81,15 @@ public void trackEvent(@NotNull String eventCategory,
         List<BasicNameValuePair> postData = Lists.newArrayList(analyticsBaseData);
         postData.add(new BasicNameValuePair("tid", analyticsId));
 
-        postData.add(new BasicNameValuePair("cd19", eventCategory));  // Event type
```

Comment:
It looks to me that the 'plugin name' and 'category' are basically passing
the same information.  In fact plugin name is better because
com.google.cloud.tools is too broad and en compasses all our plugins.

I think your strategy is good, but if we're ignoring 'category' shouldn't
we remove it as a parameter to the event call?

On Thu, Jun 23, 2016 at 10:32 AM, Chanseok Oh <notifications@github.com>
wrote:

> In
> google-account-plugin/src/com/google/cloud/tools/intellij/stats/GoogleUsageTracker.java
> <https://github.com/GoogleCloudPlatform/gcloud-intellij/pull/754#discussion_r68245423>
> :
>
> > @@ -81,17 +81,15 @@ public void trackEvent(@NotNull String eventCategory,
> >          List<BasicNameValuePair> postData = Lists.newArrayList(analyticsBaseData);
> >          postData.add(new BasicNameValuePair("tid", analyticsId));
> >
> > -        postData.add(new BasicNameValuePair("cd19", eventCategory));  // Event type
>
> I see some options to chose, since the requirement is just that the url
> should be /virtual/eventType/eventName.
>
>    1. Have eventType some kind of high-level category like
>    /virtual/appengine/deploy-success (dot or dash doesn't matter.)
>    /virtual/cloud-debugger/create-breakpoint
>
> In this case, there is no room to specify that it is from our plugin.
> That's very inconvenient. The backend team says we could pass our plugin
> name as metadata, but that still sounds inconvenient.
>
>    1.
>
>    Set com.google.cloud.tools to eventType (we won't use
>    .getExternalPluginName().)
>    /virtual/com.google.cloud.tools/appengine.deploy
>    2.
>
>    Ignore category completely
>    /virtual/gcloud-intellij/appengine.deploy
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/GoogleCloudPlatform/gcloud-intellij/pull/754/files/a9c11a4f5a3db7e0fb75e48bff6e987e090f46ec#r68245423>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AHf5HTwmCKeo6ZNYuctOUCjLXyz93Snvks5qOpkYgaJpZM4I8SzV>
> .
>